frontend/components/modals/channel_form_modal.jsx

```diff
@@ -34,8 +35,15 @@ class ChannelFormModal extends React.Component {
     this.props.formAction(this.state, this.props.selectedUsers);
   }
 
+  disabledStatus(){
+    if (!this.state.name){
```

Comment:
make this if/return a single line. will make this function so much cleaner. and actually, you should go through your app looking for if/returns in three lines, that can be made into 1 line

Reply:
I took your advice and went through making one-line if/returns as well as making sure to extract all relevant variables from this.props to make my code look cleaner.